tests/Unit/DataStructures/Test_Matrix.py

```diff
@@ -40,5 +40,13 @@ def assignment_test():
         self.assertRaises(RuntimeError, assignment_test)
 
 
+    def test_to_numpy_array(self):
+        M = Matrix(2,2)
+        M[0,0] = 1.0
+        M[0,1] = 2.0
+        A = M.to_numpy_array()
+        self.assertEquals(A[0,0], 1.0)
+        self.assertEquals(A[0,1], 2.0)
+        self.assertEquals(numpy.sin(A)[0,0], numpy.math.sin(1))
```

Comment:
Could you please add a couple blank lines before the `if __name__ == '__main__':` statement?